lcli/src/main.rs

```diff
@@ -440,6 +440,14 @@ fn main() {
                         .required(true)
                         .help("The directory in which to create the network dir"),
                 )
+                .arg(
+                    Arg::with_name("genesis-fork-version")
```

Comment:
It appears this is `required` but doesn't have that attribute in the CLI?

Reply:
Thanks, fixed it.